webmultimediadownload.cc

```diff
@@ -13,7 +13,9 @@ redirectCount( 0 )
   connect( &mgr, SIGNAL(finished(QNetworkReply*)),
            this, SLOT(replyFinished(QNetworkReply*)), Qt::QueuedConnection );
 
-  reply = mgr.get( QNetworkRequest( url ) );
+  QNetworkRequest request( url );
+  request.setAttribute(QNetworkRequest::CacheLoadControlAttribute, QNetworkRequest::PreferCache);
+  reply = mgr.get(request);
```

Comment:
This code shouldn't be necessary for the cache to work. And it risks showing the user stale, out-of-date data. The default value [`QNetworkRequest::PreferNetwork`](https://doc.qt.io/qt-5/qnetworkrequest.html#CacheLoadControl-enum) works for MediaWiki/Wikipedia.